js/binance.js

```diff
@@ -1146,6 +1146,9 @@ module.exports = class binance extends Exchange {
                     'JPY': true,
                     'NZD': true,
                 },
+                'reverseCurrencyToLeagalMoney': {
```

Comment:
we have a typo here, and the name does not seem very clear. Maybe something like `legalMoneyCurrenciesById` ?

Reply:
@carlosmiei that make more sense